src/main/views/govukTemplate.njk

```diff
@@ -14,17 +14,23 @@
 
   <!--[if IE 8]><link href="{{ asset_paths['style_vendor'] }}/fonts-ie8.css?{{ govuk_template_version }}" media="all" rel="stylesheet" /><![endif]-->
   <!--[if gte IE 9]><!--><link href="{{ asset_paths['style_vendor'] }}/fonts.css?{{ govuk_template_version }}" media="all" rel="stylesheet" /><!--<![endif]-->
+  <!--[if lt IE 9]><script src="{{ asset_paths['js_vendor'] }}javascripts/ie.js?{{ govuk_template_version }}"></script><![endif]-->
```

Comment:
javascripts is the wrong path.
This is probably in your `scripts.njk` file, and also probably has the same problem I fixed it in my PR